.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
add `.DS_Store` for mac users :)

Reply:
Done in d2cf380. I've also added the Thumbs.db 😎 😆